### DISCUSSION TITLE
Congratulations and thanks for your new version

### DISCUSSION THREAD

**Discussion by artscoop** [aclap-dev/video-downloadhelper#547] [Help and Q&A]
### Your question

I am sorry to fill the ticket stream with this, but when I saw the new design and logo of your extension, I thought "wow, I've been waiting for this for so long". Sometimes I think it's cool to go out of your way to say nice things. And it's still my go-to extension after all those years. Congratulations again.

### Addon & CoApp version and details

No extra information to provide.

**Comment by paulrouget** [comment]
Oh thank you so much ❤️  :)
